cdn/dovetail-cdn.yml

```diff
@@ -230,17 +334,28 @@ Resources:
         HttpVersion: http2
         IPV6Enabled: true
         Logging:
-          # TODO: non us-east-1 bucket
           Bucket: prx-dovetail.s3.amazonaws.com
           IncludeCookies: false
-          Prefix: !Sub "dt3-logs-${EnvironmentTypeAbbreviation}"
+          Prefix: !Sub "${AWS::StackName}-logs"
         Origins:
-          # TODO: multiple dovetail-cdn-arranger S3 bucket origins, in case
-          # lambda executions fail for some reason
           - DomainName: !Ref OriginBucket
             Id: dovetail3-stitch-s3
             S3OriginConfig:
               OriginAccessIdentity: !Sub "origin-access-identity/cloudfront/${CloudFrontOriginAccessIdentity}"
+          - !If
```

Comment:
https://github.com/PRX/Infrastructure/blob/master/CONTRIBUTING.md#function-syntax

Preferred style is `Fn::If:` in this case